meshtastic/tests/test_node_runtime_settings.py

```
@@ -0,0 +1,1311 @@
+                ota_file_hash=b"hash",
+                extra_kwargs={},
+            )
+
+    @pytest.mark.unit
+    def test_start_ota_no_mode_raises_type_error(
+        self, mock_local_node_for_admin: MagicMock
+    ) -> None:
+        """start_OTA without mode raises TypeError."""
+        runtime = _NodeAdminCommandRuntime(mock_local_node_for_admin)
+
+        with pytest.raises(TypeError, match="missing required argument.*mode"):
```

Ruff (0.15.6)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)